docs/topics/what-is-the-toolkit.adoc

```diff
@@ -11,14 +13,14 @@
 [discrete]
 == What is the {ProductName}?
 
-The {ProductName} ({ProductShortName}) is an extensible and customizable rule-based tool that simplifies the migration of Java applications.
+The {ProductName} ({ProductShortName}) is an extensible and customizable rule-based tool that simplifies the migration and modernization of Java applications.
 
-The {ProductShortName} examines application artifacts, including project source directories and application archives, then produces an HTML report that highlights areas needing changes. {ProductShortName} can migrate Java applications from earlier versions of Red Hat JBoss Enterprise Application Platform or from other application servers, such as Oracle WebLogic Server or IBM WebSphere Application Server.
+The {ProductShortName} examines application artifacts, including project source directories and application archives, then produces an HTML report that highlights areas needing changes. The {ProductShortName} supports many migration paths such as different application servers to Red Hat JBoss Enterprise Application Platform, application containerization and cloud readiness, Spring Boot to Quarkus, updates to OpenJDK, and others. For more information about the available migration paths, link:https://developers.redhat.com/products/mta/use-cases[visit the use cases and migration paths page] of the official {ProductName} website.

```

Comment:
`The {ProductShortName}`: I prefer just `MTA` rather than `the MTA`. I think the article looks odd with acronyms. If you find something in IBM that covers it, let me know.

`The {ProductShortName} supports many migration paths such as`: This is a very long sentence. I recommend breaking it up. Maybe a bullet list introduced with `The {ProductShortName} supports many migration paths including the following examples:`.

`For more information about the available migration paths, link:https://developers.redhat.com/products/mta/use-cases[visit the use cases and migration paths page] of the official {ProductName} website.` 

Change to `For more information, see link:https://developers.redhat.com/products/mta/use-cases[Migration Toolkit for Applications use cases and migration paths].`